app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ActiveRecord::Base
+  has_many :cards
+  validates_presence_of :email, :password, presence: true
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.